src/privacy-policy.md.njk

```diff
@@ -5,52 +5,105 @@ layout: layout-single-page-prose.njk
+
+## Where your data is stored
 
-We may pass on your personal information if we have a legal obligation to do so, or if we have to enforce or apply our [terms of use](https://www.gov.uk/help/terms-conditions) and other agreements. This includes exchanging information with other government departments for legal reasons.
+We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s processed and when it’s stored. 
 
-We will not share your information with any other organisations for marketing, market research or commercial purposes, and we do not pass on your details to other websites.
+Your personal data may be transferred outside the United Kingdom while being processed by GDS. If this happens, we’ll make sure you’re given the same level of technical and legal protection as you are within the United Kingdom.
+
+## Who we share your data with 
+
+As part of [GOV.UK Design System](https://design-system.service.gov.uk/) we share your personal data with data processors who provide us with:
+- software collaboration platforms when you share research, feedback or make a contribution
+- mailing list providers when you sign up to receive emails from us
+- support providers when you contact us for assistance
+
+We will not:
+- sell or rent your data to third parties
+- share your data with third parties for marketing purposes
+
+We will share your data if we’re required to do so by law — for example, by court order, or to prevent fraud or other crime.
+
+## How we protect your data and keep it secure
+We are committed to doing all that we can to keep your data secure. We set up systems and processes to prevent unauthorised access to or disclosure of the data 
+we collect about you – for example, we protect your data using varying levels of encryption. All third parties that process personal data for GDS are required to keep that data secure.
 
 ## Your rights
 
-You can [find out what information we hold about you](https://www.gov.uk/data-protection), and [ask us not to use any of the information we collect](https://www.gov.uk/contact).
+You have the right to request:
+information about how your personal data is processed, and to request a copy of that personal data
+- that any inaccuracies in your personal data is rectified without delay
+- that any incomplete personal data is updated - you can include the missing information in your request
+- that your personal data is erased if there is no longer a justification for it to be processed
+- that the processing of your personal data is restricted in certain circumstances - for example, where accuracy is contested
+
+If you gave your consent for us to collect and process your data, you have the right to:
+- withdraw your consent - this can be done at any time
+- request a copy of your personal data - this copy will be provided in a structured, commonly used and machine-readable format
```

Comment:
```suggestion
- withdraw your consent — this can be done at any time
- request a copy of your personal data — this copy will be provided in a structured, commonly used and machine-readable format
```